### PR TITLE
Refactor email is verified bouncer

### LINKED ISSUE #556: Refactor adonis hasConfirmedEmail bouncer to hasVerifiedAccount bouncer
This boucner should be applied as the same place as before
But now he should also look for a related googleID that could pass the bouncer such a hasConfirmedEmail would

### PR DESCRIPTION
**Pull request by prastoin**
close #556

### CODE REVIEW

**Review by Devessier — APPROVED** [review]
Great job!
I asked a little question.
As we discussed, we should determine if we want to test deeper authorization edge cases for HTTP routes.
We can merge.

File: packages/server/app/Controllers/Http/MyProfileController.ts [hunk 1]
```diff
@@ -13,20 +14,15 @@ export default class MyProfileController {
...
 
         /**
          * A user that has signed up using a google account won't have to verify her email
          * A user that has signed up using a mail + pswd will have to verify her email to be able to
          * later link a google account to her account
          */
-        const hasVerifiedAccount =
-            googleID !== null || confirmedEmailAt !== null;
+        const bouncer = Bouncer.forUser(user);
+        const hasVerifiedAccount = await bouncer.allows('hasVerifiedAccount');
```
[the remark this answers is not in the document]
Comment (prastoin) [reply packages/server/app/Controllers/Http/MyProfileController.ts:25]: Ah yes totally ! thought was inside a service

**Review by Devessier — APPROVED** [review]
Great job!
We can merge.
